baselines/bench/monitor.py

```diff
@@ -160,3 +162,27 @@ def load_results(dir):
     df['t'] -= min(header['t_start'] for header in headers)
     df.headers = headers # HACK to preserve backwards compatibility
     return df
+
+def test_monitor():
```

Comment:
given that the file name does not start with `test_` or end with `_test.py`, I think this test will not be found when running something like `pytest .` (for instance, travis will not actually trigger this test)

Reply:
Ah I see you moved it to another folder :-)
Ok removing it for now...I shall add all missing tests in another pr.